dev-cpp/yaml-cpp/yaml-cpp-0.7.0-r2.ebuild

```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2022 Gentoo Authors
```

Comment:
Don't commit r2 straight to stable, need to add ~ to keywords first.

Therefore, r1 shouldn't be removed yet.